common/utils/grammar.ts

```diff
@@ -9,10 +9,14 @@ export function camelize(title: string): string {
   const titleArray = title.toLowerCase().split(/-|_| /);
 
   return titleArray.reduce((acc, val, i) => {
+    const alphanumericVal = val.replace(/\W/g, '');
```

Comment:
this `camelize` function is used only as an id minter and in `content/webapp/pages/events/[eventId]/index.tsx`.
It made sense to me to remove non alphanumerical value for both these cases (otherwise it returned something like `britishSignLanguage(online)` instead of `britishSignLanguageOnline`.